app/views/console/databases/index.phtml

```diff
@@ -16,9 +16,9 @@
                 data-service="databases.list"
                 data-event="load,databases.create,databases.update,databases.delete"
                 data-param-search="{{router.params.search}}"
-                data-param-limit="<?php echo APP_PAGING_LIMIT; ?>"
-                data-param-offset="{{router.params.offset}}"
-                data-param-order-type="ASC"
+                data-param-queries="orderAsc(''),limit(<?php echo APP_PAGING_LIMIT; ?>),offset({{router.params.offset|orZero}})"
```

Comment:
Do we need to pass empty string to `orderAsc()`?

Reply:
Not required, seems to be default behaviour. Removing it.